python.d/ntp.chart.py

```diff
@@ -0,0 +1,434 @@
+# -*- coding: utf-8 -*-
+# Description: ntp netdata python.d module
+# Author: Sven Mäder (rda0)
+
+import socket
+import struct
+import re
+
+from itertools import cycle
+from base import SocketService
```

Comment:
```python
from bases.FrameworkServices.SimpleService import SimpleService
```
[base.py](https://github.com/firehol/netdata/blob/master/python.d/python_modules/base.py) will be deleted in the future